src/routes/ComparisonsRouter.ts

```diff
@@ -1,20 +1,29 @@
 import express from 'express';
 import pg from '../database/knex';
 import { Comparison } from '@src/models/Comparisons';
+import ComparisonsService from '@src/services/ComparisonsService';
 
 
 const comparisonRouter = express.Router();
 
 // Define routes
 comparisonRouter.get('/', async function (req, res) {
-  const comparisons: Comparison[] = await pg('comparisons').select('*');
+  const comparisons: Comparison[] = await ComparisonsService.getAllComparisons();
   res.send(comparisons);
 });
 
 comparisonRouter.get('/:id', async function (req, res) {
-  const comparisonId = req.params.id;
-  const comparison: Comparison = await pg('comparisons').where({id: comparisonId}).first('*');
+  const comparisonId = parseInt(req.params.id);
+  const comparison: Comparison = await ComparisonsService.getComparisonById(comparisonId);
   res.send(comparison);
 });
 
+comparisonRouter.post('/:firstId/:secondId/:voteId', async function (req, res) {
+  const firstId = parseInt(req.params.firstId);
+  const secondId = parseInt(req.params.secondId);
+  const voteId = parseInt(req.params.voteId);
+  const comparison: Comparison = await ComparisonsService.incrementComparisonByFirstSubjectIdAndSecondSubjectId(firstId, secondId, voteId);
```

Comment:
This endpoint needs to update the wins and losses for each subject too right?

Or did you plan on adding that in a future PR?

Reply:
Yeah that hasn't been added. I guess it depends on what data the UI sends back to us. But won't figure that out in this PR